draft-ietf-quic-recovery.md

```diff
@@ -412,7 +412,7 @@ is available, or if the network changes, the initial RTT SHOULD be set to 100ms.
 When an acknowledgement is received, a new RTT is computed and the timer
 SHOULD be set for twice the newly computed smoothed RTT.
 
-When crypto packets are sent, the sender MUST set a timer for the crypto
+Every time a crypto packets is sent, the sender MUST set a timer for the crypto
```

Comment:
I'm not sure this is significantly clearer, but I do understand what you're trying to address.

```suggestion
Every time a crypto packet is sent, the sender MUST set a timer for the crypto
```